conda/recipes/librmm/meta.yaml

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2018-2025, NVIDIA CORPORATION.
 
 {% set version = environ['RAPIDS_PACKAGE_VERSION'].lstrip('v') %}
+{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
```

Comment:
```suggestion
```

It doesn't look like this is getting used anywhere, maybe it's left over from some earlier version of these changes?

Reply:
I think this might've been added for consistency with other repos, but I agree let's not add it unless needed.

Reply:
I added this because I was trying to constrain rapids-logger's version, but then I realized that I was going to SemVer rapids-logger and forgot to remove it. I can delete it now.

Reply:
Done.